src/main/java/com/ylysenkova/movieland/config/TransactionManagerConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.ylysenkova.movieland.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+
+import javax.sql.DataSource;
+
+
+@Configuration
+public class TransactionManagerConfig {
+
+    @Autowired
+    private DataSource dataSource;
+    @Bean(name = "transactionManager")
```

Comment:
By default Bean will be created with name of a method. In this case no need to define name of the Been explicitly